config.json

```diff
@@ -168,6 +168,18 @@
           "label": "Config.PAYONEPAYONEINVOICESECUREDescriptionLabel",
           "options": {}
         },
+        "PAYONE_PAYONE_INVOICE_SECURE.portalid": {
+          "type": "inputText",
+          "required": false,
+          "label": "Config.portalidLabel",
+          "options": {}
+        },
+        "PAYONE_PAYONE_INVOICE_SECURE.key": {
+          "type": "inputText",
+          "required": false,
+          "label": "Config.keyLabel",
+          "options": {}
+        },
```

Comment:
Ich denke, das man für SecureInvoice ein eignes Portal und Key benötigt sollte man dann noch mal in die Doku aufnehmen.

Reply:
Das wäre dann was für @jawendel ? Auch der Ablauf, dass die Rechnungen runtergeladen und an die Aufträge gespeichert werden, sollte ja eigentlich irgendwie dokumentiert werden.

Reply:
Ja, das wäre etwas, das aufgenommen werden sollte. Ich habe mich mit dem Thema aber noch nicht auseinandergesetzt, deswegen kann ich noch nicht mehr dazu sagen.

Reply:
Hab mal eine Karte dafür aufgenommen: https://plentymarkets.kanbanize.com/ctrl_board/75/cards/119127/details/

Reply:
Sehr gut, danke.